.travis.yml

```diff
@@ -19,16 +22,15 @@ jobs:
   include:
     - stage: validate
       script:
-        - pre-commit autoupdate
-        - pre-commit run --all-files
-        - sam build
-#        - python -m pytest tests/ -v
+        - pipenv run pre-commit run --all-files
+        - pipenv run sam build
+        - pipenv run python -m pytest tests/ -v
```

Comment:
You need to either remove calls for tests or add some actual tests; it's causing [the build to fail](https://travis-ci.com/github/Sage-Bionetworks-IT/lambda-mongo-health-check/builds/183594415)

Reply:
fixed